snowplow-tracker.gemspec

```diff
@@ -9,33 +9,34 @@
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
 
-# Author::    Alex Dean, Fred Blundun (mailto:snowplow-user@googlegroups.com)
-# Copyright:: Copyright (c) 2013-2014 Snowplow Analytics Ltd
+# Author::    Snowplow Analytics Ltd
+# Copyright:: Copyright (c) 2013-2021 Snowplow Analytics Ltd
 # License::   Apache License Version 2.0
 #
 # -*- encoding: utf-8 -*-
 
-lib = File.expand_path('../lib', __FILE__)
-
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'snowplow-tracker/version'
+require_relative 'lib/snowplow-tracker/version'
 
 Gem::Specification.new do |s|
   s.name        = 'snowplow-tracker'
   s.version     = SnowplowTracker::VERSION
-  s.date        = File.mtime('./lib/snowplow-tracker/version.rb').strftime('%Y-%m-%d')
   s.homepage    = 'http://github.com/snowplow/snowplow-ruby-tracker'
-  s.license     = 'Apache License 2.0'
-  s.summary     = "Ruby Analytics for Snowplow"
-  s.description = "With this tracker you can collect event data from your Ruby applications, Ruby on Rails web applications and Ruby gems."
-  s.authors     = ["Alexander Dean", "Fred Blundun"]
+  s.license     = 'Apache-2.0'
+  s.summary     = 'Ruby Analytics for Snowplow'
+  s.description = 'With this tracker you can collect event data '\
+                  'from your Ruby applications, Ruby on Rails web'\
+                  ' applications and Ruby gems.'
+  s.authors     = ['Alexander Dean', 'Fred Blundun']
```

Comment:
Should this be `Snowplow Analytics Ltd` as well?

Reply:
Yeah, I think so.